presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -430,7 +438,8 @@ public boolean rename(Path src, Path dst)
             deleteObject(keyFromPath(src) + DIRECTORY_SUFFIX);
         }
         else {
-            s3.copyObject(getBucketName(uri), keyFromPath(src), getBucketName(uri), keyFromPath(dst));
+            s3.copyObject(new CopyObjectRequest(getBucketName(uri), keyFromPath(src), getBucketName(uri), keyFromPath(dst))
+                    .withRequesterPays(requesterPaysEnabled));
```

Comment:
Can we set this on the client rather than for every request?

Reply:
I can't find any option in AmazonS3Client to either set requester pays or add custom headers directly for the client. We have to add it for every request.